wish-list.html

```diff
@@ -0,0 +1,327 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/506bc6d1ac.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="main.css" />
+    <title>Petmate</title>
+  </head>
+  <body>
+    <div class="page-container">
+      <header class="header">
+        <div class="header__wrapper">
```

Comment:
You can change div to nav to make code more semantic.

Reply:
sure. Thank you.